style.css

```diff
@@ -0,0 +1,207 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body{
+    background-color: #faf9f7
+}
+
+#nav-bar {
+  display: flex;
+  justify-content: space-between;
+  /* align-content: space-around; */
+  /* background-color: aqua; */
+  padding: 10px;
+  background-color: white;
+  }
+
+.bar-icon {
+    margin-top: 0px;
+    font-size: 35px;
+}
+
+.icons i {
+  /* display: inline-block; */
+  padding-right: 30px;
+  font-size: 25px;
+}
+input {
+  width: 600px;
+}
+
+#search-btn {
+  width: 50px;
+}
+
+#logo {
+    margin-top: 0;
+  
+}
+.image-logo {
+  max-width: 100%;
+  width: 129px;
+  margin-top: 20px;
+}
+
+#logo span {
+  /* display: flex;
+  flex-direction: row;  */
+  /* display: inline-block; */
+  padding: 1.5rem; 
+  font-size: 25px;
+}
+
+.search-bar {
+  margin-top: 30px;
+  margin-bottom: 10px;
+  height: 20px;
+  }
+
+.icons {
+  margin-top: 30px;
+ 
+}
+
+/* #box {
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+#box1 {
+  width: 65%;
+}
+#box2 {
+  width: 30%;
+} */
+iframe {
+  width: 100%;
+  height: 70vh;
+}
+h3 {
+  font: 18px Roboto Ariel sans-serif;
+  
+}
+
+.section-icon {
+  font: 14px sans-serif;
+  display: flex;
+
+}
+
+.inside-section-icon {
+    margin: 10px;
+    margin-top: 2px;
+}
+
+.bottom-icons {
+  display: flex;
+}
+
+#Avatar {
+  border-radius: 50%;
+  height: 70px;
+  width: 70px;
+}
+
+#main-box{
+    padding-top: .1rem;
+    margin: 50px 30px;
+}
+
+#main-section {
+  float: left;
+  width: 727px;
+  padding-left: 70px;
+}
```

Comment:
- [ ] Check the width of this section. It is always best to use % instead of px when setting a width, that way you won't have problems (at least not much) when the screen size changes.

- [ ] The video should be wider and occupy around 60% of your screen.